docs/content_management/content_model.md

```diff
@@ -151,6 +151,34 @@ When a Content Type is modified, each of its instances will be changed as well.
 If a new Field definition is added to a Content Type, this Field will appear (empty) in every relevant Content item.
 If a Field definition is deleted from the Content Type, all the corresponding Fields will be removed from Content items of this type.
 
+### Removing Content Types
+
+System Content Types are by default used for the File Uploads and removing them will cause errors.
+
+If you decide to remove a `file` or `image` Content Type, or change their identifiers
+you will need to change the configuration, so it reflects the available Content Types.
+
+Example configuration:
+
+```yaml
+parameters:
+    ibexa.multifile_upload.location.default_mappings:
+        # Image
+        ...
+        content_type_identifier: custom_image_contenttype
+        content_field_identifier: image
+        name_field_identifier: name
+        # File
+        ...
+        content_type_identifier: custom_file_contenttype
+        content_field_identifier: file
+        name_field_identifier: name
```

Comment:
```suggestion
        # Image
        - mime_types:
            - image/jpeg
            - image/jpg
            - image/pjpeg
            - image/pjpg
            - image/png
            - image/bmp
            - image/gif
            - image/tiff
            - image/x-icon
            - image/webp
          content_type_identifier: custom_image_contenttype
          content_field_identifier: image
          name_field_identifier: name
        # File
        - mime_types:
            - image/svg+xml
            - application/msword
            - application/vnd.openxmlformats-officedocument.wordprocessingml.document
            - application/vnd.ms-excel
            - application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
            - application/vnd.ms-powerpoint
            - application/vnd.openxmlformats-officedocument.presentationml.presentation
            - application/pdf
          content_type_identifier: custom_file_contenttype
          content_field_identifier: file
          name_field_identifier: name
```
I think it is better to provide a full working example instead of `...` - this also makes correct indent and spacing clearer